scripts/contentScript.js

```diff
@@ -558,12 +567,105 @@ const bitBucketHunkHighlight = (apiResponses) => {
+	console.log('elementID')
+	// const rowElement = document.getElementById(`merge_request_${elementId}`);
+	const rowElement = document.getElementById(dummy)
+	if(rowElement){
+		rowElement.style.backgroundColor = backgroundColor;
+		infoElement = document.createElement('div');
+		infoElement.style.backgroundColor = tagBackgroundColor;
+		infoElement.style.color = 'green';
+		infoElement.style.width = '12px';
+		infoElement.style.height = '12px';
+		infoElement.style.border = 'rgb(45, 0, 0)';
+		infoElement.style.borderRadius = '5px';
+		infoElement.style.marginRight = '10px';
+		infoElement.style.paddingLeft = '5px';
+		infoElement.style.paddingRight = '5px';
+		infoElement.style.paddingBottom = '2px';
+		infoElement.style.textAlign = 'center';
+		infoElement.style.fontSize = '10px';
+		infoElement.style.fontWeight = 'bold';
+		infoElement.style.fontColor = 'green';
+		infoElement.innerText = 5;
+		rowElement.insertBefore(infoElement, rowElement.firstChild);
+
+		// const element = document.head.appendChild(document.createElement("style"));
+		// // TODO: a better approach would be create a constant CSS for a class, and add the class to the elements in consideration
+		// element.innerHTML = `#merge_request_230318557{
+		// background-color:${tagBackgroundColor};
+		// content: heeellogitlab;
+		// color: white;
+		// width: 12px;
+		// height: 12px;
+		// border: rgb(45, 0, 0);
+		// border-radius: 5px;
+		// margin-right: 10px;
+		// padding-left: 5px;
+		// padding-right: 5px;
+		// padding-bottom: 2px;}`;
+	}
+}
+
+
+function gitlabHunkHighlight(response){
+	
+}
+
+function highlightRelevantMRs(highlightedMRIds){
+	if(highlightedMRIds){
+		for(const priorityLevel in highlightedMRIds){
+			for (const mrNumber in highlightedMRIds[priorityLevel]){
+				addingCssElementToGitLab(mrNumber, keyToLabel[priorityLevel], highlightedMRIds[priorityLevel][mrNumber])
+			}
+		}
+	}
+};
+
+
+function showImpFileInMR(response)
+{
+	console.log('showImpFileInMR');
+	if ("relevant" in response) {
+		const fileNames = new Set(response['relevant'])
+		setTimeout(function () {
+		console.log('DOMContentLoaded');
+		const fileElements = document.querySelectorAll('div[data-qa-selector="file_row_container"]');
+		console.log(fileElements);
+		fileElements.forEach((e)=>{
+			file = e.getAttribute('data-qa-file-name')
+			// if(file==='sometext'){
+			// 	p = e.parentNode;
+			// 	p.style.backgroundColor = 'yellow';
+
+			// }
+			if(fileNames.has(file)){
+				p = e.parentNode;
+				p.style.backgroundColor = 'yellow';
+			}
+			console.log(file);
+		})
+		},1000);
+	}
+
+}
+
+
+
 
 const orchestrator = (tabUrl, websiteUrl, userId) => {
 	console.debug(`[vibinex-orchestrator] updated url: ${tabUrl}`);
 	const urlObj = tabUrl.split('?')[0].split('/');
-	if (!userId && (urlObj[2] === 'github.com' || urlObj[2] === 'bitbucket.org')) {
-		console.warn(`[Vibinex] You are not logged in. Head to ${websiteUrl} to log in`);
+
+	console.log(urlObj)
```

Comment:
Before you merge, remove all the `console.log` statements. 

Delete the statements that you use for debugging locally. And for the log statements that would be important even in production, use a better log level indicator, like `console.debug`, `console.info`, `console.warn` or `console.error`.